CHANGELOG.rst

```diff
@@ -80,6 +80,24 @@ TBR
 * Moved some of the utility functions from the test module into
   ``scrapy_poet.utils.testing``.
 
+* Supports the new :func:`scrapy.http.request.NO_CALLBACK` introduced in
+  **Scrapy 2.8**. This means that the :ref:`pitfalls` (introduced in
+  ``scrapy-poet==0.7.0``) when using the ``parse()`` method won't apply if
+  :class:`scrapy.Request <scrapy.http.Request>`'s callback is set to
+  :func:`scrapy.http.request.NO_CALLBACK`.
```

Comment:
I wonder if it'd be clear for the users what it means. Previously, we were saying "rename your parse methods". Now we say that "in Scrapy 2.8 there are no pitfalls if callback is set to NO_CALLBACK". If I'm reading it, I might think that I need to set the callback to NO_CALLBACK somewhere to avoid pitfalls, but I have no idea what is NO_CALLBACK, and when to use it.

Is it possible to say that if you use Scrapy 2.8, then pitfalls don't apply, unless you use third-party middlewares which send Requests directly to downloader and which are not updated to use NO_CALLBACK? 

I think this applies to the pitfalls doc as well. 

In Scrapy 2.8+ the pitfall is this: everything works as usual if you write spiders. But if there is some tricky code which adds Requests to downloader directly without a callback, it must use NO_CALLBACK, not None.

In Scrapy < 2.8 - pitfalls are explained in the existing pitfalls doc.

Reply:
Addressed in 7058bba